test/fetch-map/layer-map.spec.ts

```diff
@@ -5,6 +5,12 @@ import {
   getLayerProps,
   _domainFromValues,
 } from '@carto/api-client';
+import {rgb} from 'd3-color';
```

Comment:
I think `d3-color` is only in the repository as an indirect dependency now, perhaps let's add it to the devDependencies? Or just in-lining a 1-3 line function here could be fine too.

Reply:
Another observation - `d3-array` is used by code only used by `domainFromValues` which is remnant from document mode (i believe)

Reply:
>is only in the repository as an indirect dependency

Ok, i misread. Good catch. I'll try to cleanup it in next PR.